src/factory.js

```diff
@@ -173,6 +189,19 @@ export class Factory {
     return tuplet;
   }
 
+  Beam(params) {
+    params = setDefaults(params, {
+      notes: [],
+      options: {
+        autoStem: false,
+      },
+    });
+
+    const beam = new Beam(params.notes, params.options.autoStem).setContext(this.context);
```

Comment:
(Note that this code is also duplicated in  https://github.com/0xfe/vexflow/pull/417 )

The inverse of the other comment, should the `params.options` exist here even though beam doesn't take an `options` object in the constructor?